docs/DeveloperGuide.md

```diff
@@ -316,21 +332,20 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 (For all use cases below, the **System** is the `RC4HDB` and the **Actor** is the `user`, unless specified otherwise)
 
 
-**Use case: Getting help with the application**
+**Use case: UC1. Getting help with the application**
 
 **MSS**
-1. User needs help in utilizing RC4HDB.
-2. User uses the help command.
-3. RC4HDB displays a message that directs the user to our user guide.
+1. User needs help related to RC4HDB.
```

Comment:
Should we add in an additional bullet point to show the action the user has taken, e.g. "User enters the help command in the GUI"? I think this is alright as we do not specify too much UI details, given that our app is expected to be optimised for keyboard users anyway.